data/praduman8435.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "Praduman Prajapati",
+  "bio": "A **MERN** stack and **DevOps** enthusiast"
```

Please add the missing comma `,` here. 

```suggestion
  "bio": "A **MERN** stack and **DevOps** enthusiast",
```